src/components/panel/panel.js

```diff
@@ -649,6 +652,8 @@ function MdPanelService($rootElement, $rootScope, $injector) {
 MdPanelService.prototype.create = function(opt_config) {
   var configSettings = opt_config || {};
 
+  angular.copy(this._defaultConfigOptions, this._config);
```

Comment:
Extend won't copy the values, just point to the other object's values. This is what I am trying to avoid.